src/datachain/model/bbox.py

```diff
@@ -1,102 +1,84 @@
+from collections.abc import Sequence
+from warnings import warn
+
 from pydantic import Field
 
 from datachain.lib.data_model import DataModel
+from datachain.model.utils import BBoxType, convert_bbox
 
 
 class BBox(DataModel):
     """
     A data model for representing bounding box.
 
+    Use `datachain.model.Yolo` or for YOLO-specific bounding boxes.
+    This model is intended for general bounding box representations or other formats.
+
     Attributes:
         title (str): The title of the bounding box.
-        coords (list[int]): The coordinates of the bounding box.
-
-    The bounding box is defined by two points:
-        - (x1, y1): The top-left corner of the box.
-        - (x2, y2): The bottom-right corner of the box.
+        coords (list[float]): The coordinates of the bounding box.
     """
 
     title: str = Field(default="")
-    coords: list[int] = Field(default=[])
+    coords: list[float] = Field(default=[])
 
     @staticmethod
-    def from_list(coords: list[float], title: str = "") -> "BBox":
-        assert len(coords) == 4, "Bounding box must be a list of 4 coordinates."
-        assert all(isinstance(value, (int, float)) for value in coords), (
-            "Bounding box coordinates must be floats or integers."
-        )
-        return BBox(
-            title=title,
-            coords=[round(c) for c in coords],
-        )
+    def from_list(coords: Sequence[float], title: str = "") -> "BBox":
```

Comment:
Leave this method for now since it is used in many places, but mark it as `deprecated`. No need in separate method if one can build this model with `__init__` method directly.

Reply:
internally?

Reply:
I mean instead of using:

```
BBox.from_list([10, 20, 30, 40])
```

customers can use:

```
BBox(coords=[10, 20, 30, 40])
```

We don't use this model anymore in the datachain source code for now (except for the examples).